### ISSUE TITLE
Missing operator SiLU

### ISSUE THREAD

**Issue by elenabettella** [onnx/onnx#4854]
# New Operator
SiLU
### Describe the operator
SiLU is the main activation function used in [YOLOV5](https://github.com/ultralytics/yolov5)

### Can this operator be constructed using existing onnx operators?
Yes, with sigmoid

```
class SiLU(nn.Module):
    # SiLU activation https://arxiv.org/pdf/1606.08415.pdf
    @staticmethod
    def forward(x):
        return x * torch.sigmoid(x)
```
### Is this operator used by any model currently? Which one?
[YOLOV5](https://github.com/ultralytics/yolov5) uses SiLU() activation function, which can be created by x = x * sigmoid(x).

### Are you willing to contribute it? (Y/N)
N

**Comment by justinchuby** [comment]
I propose that we implement it as a function and do not take it into the spec.
